com.gopeed.Gopeed.yaml

```
@@ -0,0 +1,54 @@
+app-id: com.gopeed.Gopeed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+rename-icon: gopeed
+rename-desktop-file: gopeed.desktop
+command: gopeed
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.freedesktop.Notifications'
+  # https://github.com/miguelpruivo/flutter_file_picker/pull/1275
+  - '--filesystem=xdg-download'
+  - '--filesystem=/tmp'
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /man
+  - /share/man
+  - /share/gtk-doc
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: gopeed
+    buildsystem: simple
+    build-commands:
+      - bsdtar --to-stdout -xf gopeed.deb data.* | bsdtar -xf -
+      - mv opt/gopeed /app/
+      - cp -a usr/share /app/
+      - chmod +x /app/gopeed/gopeed
+      - install -Dm755 gopeed.sh /app/bin/gopeed
+      - install -Dm644 com.gopeed.Gopeed.metainfo.xml -t /app/share/appdata  
```

```suggestion
      - install -Dm644 com.gopeed.Gopeed.metainfo.xml -t /app/share/metainfo  
```

This is the new path https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/#path-and-filename

---

I'm sorry, I thought Flutter didn't support TMPDIR env until I found https://github.com/flutter/plugins/pull/4218